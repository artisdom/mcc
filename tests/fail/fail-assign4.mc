int main() {
  return (0 = 1); /* Error: cannot assign to an int */
}
