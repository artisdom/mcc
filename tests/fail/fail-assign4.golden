Cannot assign 1 to 0